### ISSUE TITLE
tab_model extracting AIC

### ISSUE THREAD

**Issue by schneiderpy** [strengejacke/sjPlot#911]
Hello Daniel

Is it possible to extract or add AIC values (from gls or glm models) as additional rows?
Thank you ...

**Comment by strengejacke** [comment]
Yes, there are two arguments, `show.aic` and `show.aicc`, that should add AIC and AICc to the table output.

**Comment by schneiderpy** [comment]
Thank you ....
